proto/ibc/lightclients/solomachine/v2/solomachine.proto

```diff
@@ -64,10 +62,10 @@ message Misbehaviour {
 // signature.
 message SignatureAndData {
   option (gogoproto.goproto_getters) = false;
-  bytes    signature                 = 1;
-  DataType data_type                 = 2 [(gogoproto.moretags) = "yaml:\"data_type\""];
-  bytes    data                      = 3;
-  uint64   timestamp                 = 4;
+  bytes  signature                   = 1;
+  bytes  path                        = 2;
+  bytes  data                        = 3;
+  uint64 timestamp                   = 4;
```

Comment:
I think this should technically be moved to `v3` for proto files since we are breaking the structure without changing the name

Reply:
I agree! Good spot!